packages/util/README.md

```diff
@@ -56,9 +56,67 @@ Read the [API docs](docs/).
 - [withdrawal](src/withdrawal.ts)
   - Withdrawal class (EIP-4895)
 
+### Buffer -> Uint8Array
+
+Starting with the Summer 2023 EthereumJS breaking release round (Util v9) all methods, constructors, constants and types of the EthereumJS libraries which took a `Buffer` instance as an input or resulted in a `Buffer` (containing) output have been updated to take in an `Uint8Array` instead and/or produce `Uint8Array` as an output.
+
+Here are some examples of the changes:
+
+```typescript
+async putContractStorage(address: Address, key: Buffer, value: Buffer): Promise<void> // StateManager, old
+async putContractStorage(address: Address, key: Uint8Array, value: Uint8Array): Promise<void> // StateManager, new
+
+hash(): Buffer // Block, old
```

Comment:
Is this applicable here since this is specifcally the `util` readme and this method reference is to `block`?

Reply:
Hmm, yeah, this was a concious decision since I wanted to give the broader monorepo wide context here. If you do not have a very strong opinion on this I would like to keep.